src/app/themes/theme.js

```diff
@@ -1,403 +1,413 @@
 import { createTheme } from '@material-ui/core/styles';
 
 const theme = createTheme({
-  root: {
```

Comment:
What happened with this file? It's _really hard_ to figure out what changed, and, especially with these global styles, I'm very leery of not knowing what changed here